oneview_redfish_toolkit/category_resource.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Python libs
+import threading
+
+
+# Globals vars:
+#   globals()['map_category_resources_ov']
+
+lock = threading.Lock()
+
+
+class CategoryResource(object):
+    def __init__(self, resource_id, resource, function):
+        self.resource_id = resource_id
+        self.resource = resource
+        self.function = function
+
+
+def init_map_category_resources():
+    globals()['map_category_resources_ov'] = dict()
```

Comment:
Is necessary be a global variable declared in `globals()`? It could be just a variable in this file, the scope will be global for the file, like is the [`lock `](https://github.com/HewlettPackard/oneview-redfish-toolkit/pull/462/files#diff-1bc252a74d1e386dc63f27f40ee11983R24) variable. 
With it, we could remove the call to `init_map_category_resources()` in the [app.py](https://github.com/HewlettPackard/oneview-redfish-toolkit/pull/462/files#diff-51b4fb3311d092f1b62ad16a70f9af65R174) file, and access the `map_category_resources_ov` directly (like [`lock`](https://github.com/HewlettPackard/oneview-redfish-toolkit/pull/462/files#diff-1bc252a74d1e386dc63f27f40ee11983R47) variable, too).